example-configs/mirage.conf

```diff
@@ -0,0 +1,68 @@
+#
+# minimal steps:
+#  create a disposable buildvm with 10GB of private space
+#  git clone https://github.com/xaki23/qubes-builder
+#  cd qubes-builder
+#  cp example-configs/mirage.conf builder.conf
+#  optional: comment out all-but-one for building separately
+#  make get-sources
+#  make install-deps
+#  make qubes-vm
+#  make template
+#
+
+
+RELEASE := 4.0
+SSH_ACCESS := 0
+GIT_BASEURL := https://github.com
+GIT_PREFIX := QubesOS/qubes-
+
+# include example-configs/qubes-os-r4.0.conf
+
+BUILDER_PLUGINS ?= 
+BUILDER_PLUGINS += builder-mirage
+BUILDER_PLUGINS += builder-rpm
+
+COMPONENTS ?=
+COMPONENTS += linux-template-builder
+# duplicates
+_temp_components := $(COMPONENTS)
+COMPONENTS += $(filter-out $(_temp_components), $(BUILDER_PLUGINS))
+
+DEBUG = 0
+VERBOSE = 0
+NO_SIGN = 1
+
+DIST_DOM0 ?= fc25
+
+# Only build templates (comment out or set to '0' to build all of Qubes).
+TEMPLATE_ONLY ?= 0
+
+TEMPLATE :=
+TEMPLATE += $(BUILDER_PLUGINS)
+
+GIT_URL_builder_mirage = $(GIT_BASEURL)/marmarek/qubes-builder-mirage.git
+GIT_URL_mirage_ssh_agent = $(GIT_BASEURL)/reynir/qubes-mirage-ssh-agent.git
+GIT_URL_mirage_firewall = $(GIT_BASEURL)/talex5/qubes-mirage-firewall.git
+BRANCH_builder_mirage = master
+BRANCH_mirage_ssh_agent = master
+BRANCH_mirage_firewall = master
+NO_CHECK = builder-mirage mirage-ssh-agent mirage-firewall
```

Comment:
builder-mirage (at least on my account) already have signed tag on top.
If others would have signed tags there too, better to include extra keys instead of disabling verification.
See [here](https://github.com/QubesOS/qubes-template-configs/blob/master/R4.0/templates-community/whonix-14.conf#L79-L87) for example.
But until then, please just remove `builder-mirage` from this list.